static/js/util/validation.js

```diff
@@ -11,14 +11,16 @@ import type { UIState } from '../reducers/ui';
 import { filterPositiveInt } from './util';
 import { HIGH_SCHOOL, EDUCATION_LEVELS } from '../constants';
 
-let handleNestedValidation = (profile, keys, nestedKey) => {
+let handleNestedValidation = (profile: any, keys, nestedKey) => {
```

Comment:
I wasn't able to get it to work with anything I tried except `any` here - I think the confusing is because it doesn't know if at runtime we'll be accessing the keys on an `EducationEntry` or on a `WorkHistoryEntry', if that makes sense.